Lib/test/_test_multiprocessing.py

```diff
@@ -3773,12 +3773,16 @@ def _attach_existing_shmem_then_write(shmem_name_or_obj, binary_data):
         local_sms.buf[:len(binary_data)] = binary_data
         local_sms.close()
 
+    def _new_shm_name(self, prefix):
+        return prefix + str(os.getpid())
+
     def test_shared_memory_basics(self):
-        sms = shared_memory.SharedMemory('test01_tsmb', create=True, size=512)
+        name_tsmb = self._new_shm_name('test01_tsmb')
+        sms = shared_memory.SharedMemory(name_tsmb, create=True, size=512)
```

Comment:
You may add an helper method which create a name and calls SharedMemory() with it. Or leave the code as it is, it's up to you.

Reply:
It is used not only with SharedMemory, but with ShareableList.

Reply:
Ah, I didn't notice. It's just a minor coding style suggestion. The code is good as it is.